docs/pages/foundations/tokens/colour.tsx

```diff
@@ -36,10 +36,11 @@ export default function TokensColorPage() {
 					title="On this page"
 					links={[
 						{ href: '#colour-palettes', label: 'Colour palettes' },
-						{ href: '#foreground', label: 'Foreground colors' },
-						{ href: '#background', label: 'Background colors' },
-						{ href: '#border', label: 'Border colors' },
-						{ href: '#system', label: 'System colors' },
+						{ href: '#foreground-colours', label: 'Foreground colours' },
+						{ href: '#background-colours', label: 'Background colours' },
+						{ href: '#border-colours', label: 'Border colours' },
+						{ href: '#system-colours', label: 'System colours' },
+						{ href: '#miscellaneous-colours', label: 'Miscellaneous colours' },
```

Comment:
- Needed to add 'Miscellaneous colours' to the table of contents
 - Renamed `color` to `colour`